### DISCUSSION TITLE
After enabling "Hide menubar icon" how do I get back to the settings?

### DISCUSSION THREAD

**Discussion by simon-friedberger** [rxhanson/Rectangle#1388] [Q&A] [answered]
I don't know how to get to the preferences other than through the menubar icon and I have disabled that. Help! :)

(Note: I considered manually editing the preference file described here https://github.com/rxhanson/Rectangle?tab=readme-ov-file#preferences-storage but it doesn't seem to exist.)

**Comment by rxhanson** [comment, marked as answer]
Launch Rectangle while it's running to open the prefs window. You can do this from Finder, or with this Terminal command:

```
open -a Rectangle
```

**Reply by simon-friedberger** [reply]
Okay, I'm sure I tried this but maybe that was open on a different desktop or something. It does indeed seem to work. Thank you!
